src/System.IO.FileSystem/tests/File/EncryptDecrypt.cs

```diff
@@ -41,7 +39,17 @@ public static void EncryptDecrypt_Read()
                 string fileContentRead = File.ReadAllText(tmpFileName);
                 Assert.Equal(textContentToEncrypt, fileContentRead);
 
-                File.Encrypt(tmpFileName);
+                try
+                {
+                    File.Encrypt(tmpFileName);
+                }
+                catch (IOException e) when (e.HResult == unchecked((int)0x80070490))
```

Comment:
Just double checking, the HResult `0x80070490` is correct?

Reply:
yep

Reply:
I have validated that the test passed on my machine where it used to fail before.